examples/41_fused_multi_head_attention/kernel_forward.h

```diff
@@ -163,7 +163,7 @@ struct AttentionKernel {
     int32_t num_heads;
 
     CUTLASS_HOST_DEVICE int32_t o_strideM() const {
-      return head_dim_value;
+      return head_dim_value * num_heads;
```

Comment:
I don't think this is correct, as we produce outputs in BHMK format in the cutlass implementation (vs BMHK in AITemplate - that's why there is a difference).
This will change to BMHK in https://github.com/NVIDIA/cutlass/pull/773 anyway :)